sql-plugin/src/main/java/com/nvidia/spark/rapids/iceberg/parquet/GpuParquet.java

```diff
@@ -67,6 +67,8 @@ public static class ReadBuilder {
     private Configuration conf = null;
     private int maxBatchSizeRows = Integer.MAX_VALUE;
     private long maxBatchSizeBytes = Integer.MAX_VALUE;
+    private long targetBatchSizeBytes = Integer.MAX_VALUE;
```

Comment:
For me, I'm very confused in differentiating between `maxBatchSizeBytes` and `targetBatchSizeBytes`.

Reply:
Do you want a comment in here? The long term plan is to eventually get rid of the maxBatchSizeBytes and the maxBatchSizeRows, but we can only do that one we have chunked reads the only option for Parquet and ORC and depending on how things go for avro, CSV, and JSON too.

Reply:
Yes please add comment here to clarify how they are different.